networks/known_networks.go

```diff
@@ -122,8 +122,8 @@ var (
 		SimulationType:       "besu",
 		ChainID:              2337,
 		PrivateKeys: []string{
+			"ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80",
```

Comment:
Hey let's make this as second key for chain id with 2337. The reason - if you use same private key for 1337 and 2337 it will create contracts on both chains with same address. It's not an issue but it becomes a little bit difficult during debugging.